.travis.yml

```diff
@@ -0,0 +1,13 @@
+sudo: false
+language: python
+python:
+- '2.7'
+- '3.2'
```

Comment:
It looks like people are slowly dropping support for 3.2, which is EOL next month, so we could drop this?